.env

```diff
@@ -1,2 +1,2 @@
 REACT_APP_PROXY_URL = https://sandbox.dissco.tech
-REACT_APP_AXIOS_URL = https://sandbox.dissco.tech/api/v1
+REACT_APP_AXIOS_URL = http://localhost:3000/api/v1
```

Comment:
Is this going to work?

Reply:
No, accidentally pushed the development variables, new commit resets them to production